app/_assets/stylesheets/pages/docs.less

```
@@ -179,6 +179,16 @@
   }
 
   &.v2 {
+    .content {
+      .breadcrumbs {
+        margin-left: -6rem;
```

```suggestion
        margin-left: -0.1rem;
```
Can you align it with the content text instead? I think this is about right, but I could be off a bit.